### PR TITLE
Add details on setting aws cli

### PR DESCRIPTION
**Pull request by chao-xian**
Because this was not in the slightest bit obvious to me when I did it and had to hassle @issyl0

### CODE REVIEW

File: source/manual/access-aws-console.html.md [hunk 1]
```diff
@@ -19,19 +19,51 @@ The (private) GitHub repo is [alphagov/gds-cli](https://github.com/alphagov/gds-
...
+Enter token for arn:aws:iam::123456789012:mfa/firstname.lastname@digital.cabinet-office.gov.uk: 123456
+```
+
+Your Access Key is from the AWS console:
+
+![AWS Access Key](images/aws/access-key.png)
+
+If you've forgotten the secret, regenerate it and save your key's secret somewhere safe!
+
+You'll also be prompted to save credentials to your Mac's Keyvault as `aws-vault` and set a password for it. Save that password somewhere safe too!
```
Comment (issyl0) [inline source/manual/access-aws-console.html.md]:
```suggestion
You'll also be prompted to save credentials to your Mac's Keychain as `aws-vault` and set a password for it. Save that password somewhere safe too, like a password manager.
```

File: source/manual/access-aws-console.html.md [hunk 2]
```diff
@@ -19,19 +19,51 @@ The (private) GitHub repo is [alphagov/gds-cli](https://github.com/alphagov/gds-
...
+
+> Note:  If you see `fatal: yubikey: no such path in the working tree`, it's because you're using ZSH, which has `gds` set up as a Git alias. You can either remove that alias by adding `unalias gds` to your `~/.zshrc`, or use `gds-cli` instead of `gds` for all the commands below.
 
 If you have `bash-completion` installed and configured, the gds-cli tab completions will work out of the box. They're especially useful for long commands like AWS account names.
 
+## Re-initialising
+
+If you have problems, like forgetting your `aws-vault` password, delete the local creds:
```
Comment (issyl0) [inline source/manual/access-aws-console.html.md]:
```suggestion
If you forget your `aws-vault` password, delete the aws-vault keychain with `rm ~/Library/Keychains/aws-vault-keychain-db` and re-initialise the `gds-cli` by changing `initialised: true` to `initialise: false` in `~/.gds/config.yml`, then re-run the "First run" commands above.
```

File: source/manual/access-aws-console.html.md [hunk 3]
```diff
@@ -19,19 +19,51 @@ The (private) GitHub repo is [alphagov/gds-cli](https://github.com/alphagov/gds-
...
+
+```shell
+rm ~/Library/Keychains/aws-vault.keychain-db
+```
+
+and re-run the "First run" section above.
+
```
Comment (issyl0) [inline source/manual/access-aws-console.html.md]: These lines can go.

File: source/manual/access-aws-console.html.md [hunk 4]
```diff
@@ -19,19 +19,51 @@ The (private) GitHub repo is [alphagov/gds-cli](https://github.com/alphagov/gds-
...
 ## Get Your Role
 
 Work out which [list of users you're part of in govuk-aws-data][govuk-aws-data-users-group].
 You're about to use it in the 'Web Console' section below.
 
 ## Usage
 
+
```
Comment (issyl0) [inline source/manual/access-aws-console.html.md]: I don't know why there's extra blank line here...

File: source/manual/access-aws-console.html.md (hunk 3 above)
Comment (chao-xian) [reply source/manual/access-aws-console.html.md]: Done - but I also reformatted your commit to be slightly more readable

**Pull request merged**